### ISSUE TITLE
jsdoc/require-jsdoc not always reporting on TypeAlias properties

### ISSUE THREAD

**Issue by sid2u** [gajus/eslint-plugin-jsdoc#852]
## Expected behavior
When `"TSPropertySignature"` is added as one of the `contexts` for the `jsdoc/require-jsdoc` rule, it should always report missing JSDoc comments on properties inside Typescript TypeAlias definitions.

## Actual behavior
The linter doesn't complain at property-level when the TypeAlias itself has JSDoc comments.

## ESLint Config
Rule configuration:
```json
    "jsdoc/require-jsdoc": [
      "warn",
      {
        "publicOnly": true,
        "require": {
          "ArrowFunctionExpression": true,
          "ClassDeclaration": true,
          "ClassExpression": true,
          "FunctionDeclaration": true,
          "FunctionExpression": true,
          "MethodDefinition": true
        },
        "contexts": [
          "TSTypeAliasDeclaration", // <-------
          "TSInterfaceDeclaration",
          "TSModuleDeclaration",
          "TSMethodSignature",
          "TSPropertySignature", // <-------
          "TSDeclareFunction",
          "TSEnumDeclaration",
          "ClassProperty[accessibility='public']",
          "ExportNamedDeclaration > VariableDeclaration"
        ]
      }
    ]
```

## ESLint sample
Sample code where the missing JSDoc comment is not reported:
```ts
/** Alpha of all types */
export type Alpha = {
  one: string;
  two: number;
};
```

Screenshots with and without comment at the type-alias level, but NO comment at property level:

|With comment|Without comment|
|---|---|
| ![image](https://user-images.githubusercontent.com/8316961/158207351-076c72da-153f-4065-a3d6-b54123d85643.png) | ![image](https://user-images.githubusercontent.com/8316961/158207721-273c89d4-0166-4ddc-a319-bde676f121c8.png) |

## Environment

- Node version: `v14.18.0`
- ESLint version: `^7.30.0`
- `eslint-plugin-jsdoc` version: `^37.9.6`

**Comment by sid2u** [comment]
Thanks for the quick turn-around on this!
